packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewerContainer.js

```diff
@@ -1,62 +1,48 @@
 import asyncStates from '@zooniverse/async-states'
 import PropTypes from 'prop-types'
-import { useEffect, useRef, useState } from 'react';
+import { useEffect, useState } from 'react';
+
+import { useSubjectImage } from '@hooks'
 
 import withKeyZoom from '../../../withKeyZoom'
 import locationValidator from '../../helpers/locationValidator'
-import useSubjectImage, { PLACEHOLDER_URL } from './hooks/useSubjectImage'
 import SVGImage from '../SVGComponents/SVGImage'
 import SVGPanZoom from '../SVGComponents/SVGPanZoom'
 import SingleImageViewer from './SingleImageViewer'
 
+const DEFAULT_HANDLER = () => true
+
 function SingleImageViewerContainer ({
   enableInteractionLayer = true,
-  enableRotation = () => null,
+  enableRotation = DEFAULT_HANDLER,
   frame = 0,
   invert = false,
   loadingState = asyncStates.initialized,
   move = false,
-  onError = () => true,
-  onKeyDown = () => true,
-  onReady = () => true,
+  onError = DEFAULT_HANDLER,
+  onKeyDown = DEFAULT_HANDLER,
+  onReady = DEFAULT_HANDLER,
   rotation = 0,
-  setOnPan = () => true,
-  setOnZoom = () => true,
+  setOnPan = DEFAULT_HANDLER,
+  setOnZoom = DEFAULT_HANDLER,
   subject,
   title = {},
   zoomControlFn,
   zooming = true
 }) {
-  const subjectImage = useRef()
   const [dragMove, setDragMove] = useState()
   // TODO: replace this with a better function to parse the image location from a subject.
   const imageUrl = subject ? Object.values(subject.locations[frame])[0] : null
-  const { img, error, loading } = useSubjectImage(imageUrl)
-  // default to a placeholder while image is loading.
-  const { naturalHeight, naturalWidth, src } = img
-
-  useEffect(function onImageLoad() {
-    if (src !== PLACEHOLDER_URL ) {
-      const svgImage = subjectImage.current
-      const { width: clientWidth, height: clientHeight } = svgImage
-        ? svgImage.getBoundingClientRect()
-        : {}
-      const target = { clientHeight, clientWidth, naturalHeight, naturalWidth }
-      onReady({ target })
-    }
-  }, [src])
+  const { img, error, loading, subjectImage } = useSubjectImage({
```

Comment:
I noticed that in all three subject viewers, `error` and `loading` returned from `useSubjectImage()` aren’t used in the component. Do you have suggestions on how to utilize those variables? For instance, will there ever be a case when `img` is undefined?

Reply:
Those are there to catch errors. Maybe if the image URL 404s, perhaps?

Reply:
I think we can test this out in the storybook by adding a mock subject with a URL that errors. 

I put examples of how to use `loading` and `error` in the Readme too, but you’re right that the viewers aren’t using them.